spring-cloud-dataflow-rest-client/pom.xml

```diff
@@ -46,6 +46,29 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-skipper</artifactId>
 			<version>${project.version}</version>
+			<!-- TODO causes tests to autoconfigure datasource when none is needed -->
```

Comment:
I am definitely not keen on doing these exclusions in this PR as I feel this could have other ripple effects. There are other ways to exclude the test database from being autoconfigured during tests. If this is getting in your way let's use something like autoconfiguration exclusion etc.. 

Really it seems like the rest client needs a lighter lib from skipper (not the kitchen sink) but lets handle that in a subsequent PR.

Reply:
Let me check if there is a better approach in terms of Skipper dependencies.

Reply:
Maybe we push this to phase 2 when we have skipper migrated.   Thoughts?

Reply:
Moved to separate PR